app/navigation/ConsignorNavigator.scala

```diff
@@ -19,14 +19,17 @@ package navigation
 import controllers.routes
 import models.{CheckMode, Mode, NormalMode, ReviewMode, UserAnswers}
 import pages.Page
-import pages.sections.consignor.{CheckAnswersConsignorPage, ConsignorAddressPage}
+import pages.sections.consignor.{CheckAnswersConsignorPage, ConsignorAddressPage, ConsignorPaidTemporaryAuthorisationCodePage}
 import play.api.mvc.Call
 
 import javax.inject.Inject
 
 class ConsignorNavigator @Inject() extends BaseNavigator {
```

Comment:
Should this also have Check/Change routes?

Reply:
`checkRouteMap` and `reviewRouteMap` are already present and their only action is to navigate to CYA which I believe is correct.